docs/source/notebooks/Tutorial.md

```diff
@@ -0,0 +1,179 @@
+
+As expected, this function returns the evolution of the environment,
+the observed results, the reward of the performed action, if the agent enters
+a terminal state, and additional information about the process.
+
+If we are not interested in getting the current state of the environment, we simply define the argument
+`return_state = False` inside the methods `plangym.Env.reset` and `plangym.Env.step`: 
+
+```python
+import plangym
+env = plangym.make(name="CartPole-v0")  
+obs = env.reset(return_state=False)  
+
+action = env.action_space.sample()
+
+data = env.step(action=action, return_state=False)
+observ, reward, end, info = data
+```
+
+By setting `return_state=False`, neither `reset()` nor `step()` will return the state of the simulation. In this way,
+we are obtaining the exact same answers as if we were working in a plain `gym` interface. Thus, `plangym`
+provides a complete tool for developing planning projects __as well as__ a general, standard API for reinforcement learning problems.  
+
+### Stepping a batch of states and actions
+```python
+import plangym
+env = plangym.make(name="CartPole-v0")
+state, obs = env.reset()
+
+states = [state.copy() for _ in range(10)]
+actions = [env.action_space.sample() for _ in range(10)]
+
+data = env.step_batch(states=states, actions=actions)
+new_states, observs, rewards, ends, infos = data
+```
+
+`plangym` allows applying multiple actions in a single call via the command `plangym.Env.step_batch`.
+The syntax used for this case is reminiscent to that employed when calling a `step` function; we should define 
+a __list__ of states and actions and use them as arguments of the function `step_batch()`. `plangym` will
+take care of distributing the states and actions correspondingly, returning a tuple with the results
+of such actions. 
+
+### Making environments
+
+To initialize an environment, `plangym` uses the same syntax as `gym` via the `plangym.make` command. However, 
+this command offers more advanced options than the `gym` standard; it controls the general behavior of the API and
+its different environments, and it serves as a command center between the user and the library. 
+
+Instead of using a specific syntax for each environment (with distinct arguments and parameters),
+`plangym` unifies all options within a single, common framework under the control of
+`make()` command.
+
+All instance attributes are defined through the `make()` command, which classifies and distributes them accordingly whether
+they belong to `plangym` or standard parameters. In addition, `make()` also allows the user to configure the
+parameters needed for stepping the environment in parallel. One only should select the desired mode, and
+`plangym` will do the rest.
+
+```python
+import plangym
+env = plangym.make(
+    name="PlanMontezuma-v0",   # name of the environment
+    n_workers=4,  # Number of parallel processes
+    state='',  # Define a specific state for the environment
```

Comment:
```suggestion
```
The `state` parameter only works for `RetroEnv`, and it's used to select the starting level of the selected game. All the other environments do no accept state as a keywork argument, and specific states can be set using `get_state` and `set_state`.